### PR TITLE
add task layout_catalog

### PR DESCRIPTION
**Pull request by volodymyrsar**
- [DEMO LINK](https://volodymyrsar.github.io/layout_catalog/)
- [TEST REPORT LINK](https://volodymyrsar.github.io/layout_catalog/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -3,10 +3,384 @@
...
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#Home">
+        <img class="logo"
+        src="./images/logo.png"
+        alt="Moyologo">
```
Comment (anstsot) [inline src/index.html:17]: please check the code styling
```suggestion
        <img
          class="logo"
          src="./images/logo.png"
          alt="Moyologo"
        >
```

File: src/index.html [hunk 2]
```diff
@@ -3,10 +3,384 @@
...
+          <p class="price__word">Price:</p>
+          <p class="price__value">$2,199</p>
+        </div>
+
+        <a
+          class="button"
+          href="#"
+          data-qa="card-hover"
+        >
+        Buy
```
Comment (anstsot) [inline src/index.html:93]: fix the tabulation
```suggestion
          Buy
```

File: src/index.html [hunk 3]
```diff
@@ -3,10 +3,372 @@
...
+    <main class="container">
+      <div
+        data-qa="card"
+        class="card"
+      >
+        <div class="card__cards">
+          <img
+            class="card__image"
+            src="./images/imac.jpeg"
+            alt="card__image monitor Apple"
```
Comment (sTorba24) [inline src/index.html:65]:
```suggestion
            alt="Apple monitor"
```
https://www.w3schools.com/tags/att_img_alt.asp

File: src/styles/Card.scss [hunk 4]
```diff
@@ -0,0 +1,100 @@
+.container {
```
Comment (sTorba24) [inline src/styles/Card.scss:1]: change first letter to lowercase in file name

File: src/styles/main.scss [hunk 5]
```diff
@@ -1,3 +1,7 @@
-body {
-  margin: 0;
-}
+@import "./variables";
+@import "./normolized.scss";
```
Comment (sTorba24) [inline src/styles/main.scss:2]:
```suggestion
@import "./normalized.scss";
```
typo

**Pull request closed without merging**